nodes/xrd/xrd.cfg

```diff
@@ -0,0 +1,50 @@
+!
+hostname {{ .ShortName }}
+!
+username clab
+ group root-lr
+ group cisco-support
+ secret clab@123
+!
+grpc
+{{ if .Env.CLAB_MGMT_VRF }} vrf {{ .Env.CLAB_MGMT_VRF }}{{end}}
+ port 9339
```

Comment:
@hellt The port number range is 57344-57999.
Ref: https://github.com/CiscoDevNet/grpc-getting-started/blob/master/README.md#grpc-server

Reply:
this must be an outdated info. IANA assigned port for gnmi/gnoi is 9339 and since it works with xrd I am gonna use it to prevent proliferation of custom port ranges